### PR TITLE
Add events committee to main branch

### PR DESCRIPTION
**Pull request by Tguntenaar**
_Resolves #842_

As part of issue #777

## Check list
- [ ] None of the deployed components to the `events-committee.near` account change.
- [x] Github Action that runs the CI for both devhub and events comitee repository.
- [ ] Docs are updated

In the next phase of issue #777 we can unify proposal tests in 1 file. By editing the playwright config projects testMatch cases like this: `/(proposal|events)\/.*.spec.js/,`. I've skipped it in this PR to closer resamble @Megha-Dev-19 her changes in issue #843.

## This PR
- Adds the code from the `events-committee` branch into the main branch.
- Edits proposals.spec for future merging of all proposal tests into a single file instead of 3 seperate once.

I found that the events-commitee is not up-to-date with the main portal so I removed those tests from the proposals. We should make those changes in another PR. (Once this one is merged to chop it in up in smaller bits)

## Adding variables

- `NEAR_EVENTS_SOCIAL_ACCOUNT_ID`
- `NEAR_EVENTS_SOCIAL_ACCOUNT_PUBLIC_KEY`
- `NEAR_EVENTS_SOCIAL_ACCOUNT_PRIVATE_KEY`
- `NEAR_EVENTS_SOCIAL_DEPLOY_ID`

# Future PR
- has to add the clean URL that @petersalomonsen added
- relevant mentions that I've added
- sync @Megha-Dev-19 her KYC updates

### CODE REVIEW

**Comment by petersalomonsen** [comment]
So as we discussed in the call, we will need:

- A separate copy of the CI workflow targeting the events committee instance ( running the tests that are relevant )
- A deployment action that does diff ( dry-run ) when not on the main branch

File: playwright-tests/tests/blog/blog.spec.js
Comment (Megha-Dev-19) [inline playwright-tests/tests/blog/blog.spec.js:3]: why are we removing this?

**Review by Megha-Dev-19 — APPROVED** [review]
LGTM, but I see some tests are failing.

File: .github/workflows/deploy-dry-mainnet-events.yml
Comment (Megha-Dev-19) [inline .github/workflows/deploy-dry-mainnet-events.yml:1]:
```suggestion
name: Deploy Widgets to Mainnet - Events
```
Just for consistency in all three workflows

File: .github/workflows/deploy-dry-mainnet-events.yml
Comment (Megha-Dev-19) [inline .github/workflows/deploy-dry-mainnet-events.yml:10]:
```suggestion
    name: Deploy ( or diff from PR )
```
Just for consistency in all three workflows

**Comment by Tguntenaar** [comment]
@race-of-sloths please, include my PR in the race!

**Comment by race-of-sloths** [comment]
@Tguntenaar Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
New Sloth joined the Race! Welcome!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/Tguntenaar?type=bot&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/Tguntenaar?type=bot&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/Tguntenaar)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command
</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/Tguntenaar)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Review by Tguntenaar — COMMENTED** [review]
@Megha-Dev-19 the tests are fixed.

**Review by petersalomonsen — APPROVED** [review]
Great @Tguntenaar Let's get this updated with the main branch, and merged as soon as possible!

**Comment by race-of-sloths** [comment]
### 🔄 The PR has been merged.
>[!IMPORTANT]
This pull request is a part of the Race of Sloths and has not been scored yet. Scoring will close in 24 hours! 🕰️

**Comment by race-of-sloths** [comment]
### ✅ PR is finalized! 
Your contribution is much appreciated with a final score of 0!
You have received 10 (0 base + 10 weekly bonus) Sloth points for this contribution


Another weekly streak completed, well done @Tguntenaar! To keep your weekly streak and get another bonus make pull request next week! Looking forward to see you in race-of-sloths

**Pull request merged**
